.circleci/config.yml

```diff
@@ -276,14 +276,14 @@ jobs:
   yaml:
     <<: *common
     docker:
-      - image: cimg/python:3.8
+      - image: cimg/python:3.13
```

Comment:
Curious on why this change. Also `yaml` is commented out in the test runs below (L313). I might be missing some context here, but should this be turned on?

Reply:
yaml tests were turned off a while ago, as noted here - https://github.com/ethereum/py-ssz/issues/137

I recall there being an issue with `ruaml.yaml` dep too, but not specifically what.

I just updated it there because I was doing the same for the benchmark tests.

Reply:
Thanks for the background. sgtm, glad we're tracking that with an issue at least 👍🏼